### PR TITLE
Blog app - controllers (PR)

### PR DESCRIPTION
**Pull request by teke85**
In this PR I fulfilled the following requirements:
231098415-06d36017-db0c-4343-865d-e4bf5948cd75

I made sure that my app handled URLs based on the wireframes above.
For each url i created:
- A route.
- An action in the correct controller.
- A view file.
- I created View files which included only placeholders written in plain HTML, e.g:
- Here is a list of posts for a given user

I also made sure that I added code to handle only the 4 required URLs.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+end
```
Comment (algerina) [inline app/controllers/users_controller.rb]: - [ ] You did well  setting the controllers however it is required to have an index and a show actions. Kindly add the index action.

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class PostsController < ApplicationController
+  def show
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:id])
+  end
+end
```
Comment (algerina) [inline app/controllers/posts_controller.rb]: - [ ] Kindly add the index action in your Posts controller.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @teke85 ,

Great job so far! 

May I highlight the following: 
- [x] Your project is professionally documented👏  ✔️
- [x] Your pull request is descriptive👏  ✔️



## CHANGES REQUIRED:muscle::clap:
Check the comments under the review.



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: app/controllers/posts_controller.rb (hunk 2 above)
Comment (teke85) [reply app/controllers/posts_controller.rb]: @algerina Hi algerina, I have added an index action in my post controller, thank you very much for reviewing my code.

**Comment by prolajumokeoni** [comment]
Hi @teke85, kindly fix the rubocop error 👍🏿

**Comment by teke85** [comment]
> Hi @teke85, kindly fix the rubocop error 👍🏿
Error has been fixed

**Review by prolajumokeoni — APPROVED** [review]
Hi @teke85 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- All changes requested have been implemented ✅ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**